frontend/src/assets/img/logos/logos.ts

```diff
@@ -0,0 +1,3 @@
+export { default as logoM } from './logo-m.svg';
```

Comment:
remove this file, no ts files in assets, also remove logos folder and just keep them in assets/img